core/src/main/java/com/scalar/db/storage/cosmos/CosmosAdmin.java

```diff
@@ -53,7 +53,8 @@ public class CosmosAdmin implements DistributedStorageAdmin {
   public static final String DEFAULT_NO_SCALING = "false";
```

Comment:
Don't we need to change the implementation of `namespaceExists()` as well?

Reply:
Ah, we may need to change `namespaceExists()` in `JdbcAdmin` as well.

Reply:
Yes, good point thank you. I will update `namespaceExists()` in all the admins.

Reply:
I revised the `namespaceExists()` implementation in the latest commit. Please have a look when you can.